src/detail-page/DetailPage.tsx

```diff
@@ -22,13 +19,22 @@ export const TabContainer = styled('div')`
     }
 `;
 
-export const DetailPage: React.FC<DetailProps> = ({
-  profession,
-}) => (
-  <MainContainer>
-    <div>{profession.title}</div>
-    <TabContainer>
-      <Tabs categories={profession.categories} learn="What to learn" jobs="jobs" resources="resources" />
-    </TabContainer>
-  </MainContainer>
-);
+export const DetailPage: React.FC<{}> = () => {
+  const { professionId } = useParams<{professionId: string}>();
```

Comment:
As mentioned above - we don't really need to pass the prop, we can take advantage of information in our params :)